dist/smart-table-react.js

```diff
@@ -128,111 +119,99 @@ const SEARCH_CHANGED = 'SEARCH_CHANGED';
 
 const filterListener = proxyListener({[FILTER_CHANGED]: 'onFilterChange'});
 
-var filterDirective = function ({table, pointer, operator = 'includes', type = 'string'}) {
-  return Object.assign({
-      filter(input){
-        const filterConf = {
-          [pointer]: [
-            {
-              value: input,
-              operator,
-              type
-            }
-          ]
-
-        };
-        return table.filter(filterConf);
-      }
-    },
-    filterListener({emitter: table}));
-};
+var filterDirective = ({table, pointer, operator = 'includes', type = 'string'}) => Object.assign({
+	filter(input) {
+		const filterConf = {
+			[pointer]: [
+				{
+					value: input,
+					operator,
+					type
+				}
+			]
+
+		};
+		return table.filter(filterConf);
+	}
+}, filterListener({emitter: table}));
 
 const searchListener = proxyListener({[SEARCH_CHANGED]: 'onSearchChange'});
 
-var searchDirective = function ({table, scope = []}) {
-  return Object.assign(
-    searchListener({emitter: table}), {
-      search(input){
-        return table.search({value: input, scope});
-      }
-    });
-};
+var searchDirective = ({table, scope = []}) => Object.assign(searchListener({emitter: table}), {
+	search(input) {
+		return table.search({value: input, scope});
+	}
+});
 
 const sliceListener = proxyListener({[PAGE_CHANGED]: 'onPageChange', [SUMMARY_CHANGED]: 'onSummaryChange'});
 
 var sliceDirective = function ({table}) {
-  let {slice:{page:currentPage, size:currentSize}} = table.getTableState();
-  let itemListLength = table.length;
-
-  const api = {
-    selectPage(p){
-      return table.slice({page: p, size: currentSize});
-    },
-    selectNextPage(){
-      return api.selectPage(currentPage + 1);
-    },
-    selectPreviousPage(){
-      return api.selectPage(currentPage - 1);
-    },
-    changePageSize(size){
-      return table.slice({page: 1, size});
-    },
-    isPreviousPageEnabled(){
-      return currentPage > 1;
-    },
-    isNextPageEnabled(){
-      return Math.ceil(itemListLength / currentSize) > currentPage;
-    }
-  };
-  const directive = Object.assign(api, sliceListener({emitter: table}));
-
-  directive.onSummaryChange(({page:p, size:s, filteredCount}) => {
-    currentPage = p;
-    currentSize = s;
-    itemListLength = filteredCount;
-  });
-
-  return directive;
+	let {slice: {page: currentPage, size: currentSize}} = table.getTableState();
```

Comment:
something looks wrong with the formatting (tab vs space or something) might be my fault though but whatever style chose should be consistent

Reply:
rollup output is well-formed by default now. Do you want to revert it?